packages/components/src/List/Toolbar/ColumnChooser/hooks/columnChooserClient.hook.js

```diff
@@ -0,0 +1,20 @@
+import { useState, useEffect } from 'react';
+import { mergedColumnsChooser } from '../service';
+
+function useColumnChooserClient(columns, handlerCustomColumnChooser = () => {}) {
```

Comment:
how about having `columns = []` in function parameters & `useState({ columns })` in the function ?